packages/security/src/components/TrendingCard/index.js

```diff
@@ -9,7 +9,7 @@ import React, { Fragment } from 'react';
 
 import { getComponentNamespace } from '../../globals/namespace';
 
-const namespace = getComponentNamespace('trending-card');
+const namespace = getComponentNamespace('trending-card#legacy');
```

Comment:
Changing the namespace looks like it also affects (breaks) the styles in Storybook here.

| Current | Expected |
---------|-----------|
![image](https://user-images.githubusercontent.com/1717398/165121728-6786d26d-8a8d-4721-aec9-c553e37c8bd5.png) | ![image](https://user-images.githubusercontent.com/1717398/165121802-dc76bf5d-706a-4f72-b786-b1bcc8b499cd.png)

Reply:
@elycheea Right! I have updated the PR with the labels on the page and also updated the Trending Card component so the styles don't break. There are three components that I have not been able to update due to the nature of the component but will continue to work on it. I do not think I want to hold up the PR for those three components and will create another. But let me know if you think differently.